coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE liquidity_requests (
+    id SERIAL PRIMARY KEY,
+    trader_pk TEXT NOT NULL,
```

Comment:
This could be a reference to `users`.`pubkey`

Reply:
great idea. Should we do similar with `trades.trader_pubkey` ?